src/style.css

```
@@ -1,3 +1,83 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap");
+
+:root {
+  --main-color: #00acdc;
+  --black: #000;
+  --background-color: #fff;
+}
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+  background-color: var(--background-color);
+}
```

Only `box-sizing` property allow for `*` selector
```suggestion
* {
  box-sizing: border-box;
}

body {
  margin: 0;
  padding: 0;
  background-color: var(--background-color);
}
```